app/services/client_messaging_service.rb

```diff
@@ -62,12 +77,9 @@ def send_message_to_all_opted_in_contact_methods(client:, user:, body:, locale:
       args = { client: client, user: user, body: body }
       args[:tax_return] = tax_return if tax_return.present?
       args[:locale] = locale if locale.present?
-      if client.intake.email_notification_opt_in_yes? && client.email_address.present?
-        message_records[:outgoing_email] = send_email(**args)
-      end
-      if client.intake.sms_notification_opt_in_yes? && client.sms_phone_number.present?
-        message_records[:outgoing_text_message] = send_text_message(**args)
-      end
+      # returns nil unless client opted in to contact method
```

Comment:
does this comment refer to what `send_email` and `send_text_message` return? if so it looks like they actually return false